addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -576,7 +576,7 @@ def _l10n_es_edi_call_web_service_sign(self, invoices, info_list):
 
             else:
                 results[inv] = {
-                    'error': _("[%s] %s", respl.CodigoErrorRegistro, respl.DescripcionErrorRegistro),
+                    'error': _("[%(error_code)s] %(error_message)s", error_code=respl.CodigoErrorRegistro, error_message=respl.DescripcionErrorRegistro),
```

Comment:
I don't think it is worth to translate this one
```suggestion
                    'error': f"[{respl.CodigoErrorRegistro}] {respl.DescripcionErrorRegistro}",
```

Reply:
What about r2l languages?